### PR TITLE
Added Danish translations

### PR DESCRIPTION
**Pull request by knsjensen**

### CODE REVIEW

File: custom_components/alarmo/frontend/localize/languages/da.json [hunk 1]
```diff
@@ -0,0 +1,554 @@
...
+      "editor": {
+        "ui_mode": "Til UI",
+        "yaml_mode": "Til YAML",
+        "edit_in_yaml": "Ret i YAML"
+      },
+      "table": {
+        "filter": {
+          "label": "Filtrer genstande",
+          "item": "Filtrer efter {name}",
+          "hidden_items": "{number} {number, plural,\n en {item is}\n  other {items are}\n} skjult"
```
Comment (nielsfaber) [inline custom_components/alarmo/frontend/localize/languages/da.json:29]: The word 'en' needs to be replaced by 'one'.
The words within brackets {item is} and {items are} should be translated instead.

File: custom_components/alarmo/frontend/localize/languages/da.json [hunk 2]
```diff
@@ -0,0 +1,554 @@
...
+            }
+          },
+          "areas": {
+            "title": "Områder",
+            "description": "Områder kan bruges til at opdele dit alarmsystem i flere rum.",
+            "no_items": "Der er ikke defineret områder endnu.",
+            "table": {
+              "remarks": "Kommentarer",
+              "summary": "Dette område indeholder {summary_sensors} og {summary_automations}.",
+              "summary_sensors": "{number} {number, plural,\n  en {sensor}\n  other {sensors}\n}",
```
Comment (nielsfaber) [inline custom_components/alarmo/frontend/localize/languages/da.json:126]: Same comment as before

File: custom_components/alarmo/frontend/localize/languages/da.json [hunk 3]
```diff
@@ -0,0 +1,554 @@
...
+              "summary_automations": "{number} {number, plural,\n  en {automation}\n  other {automations}\n}"
```
Comment (nielsfaber) [inline custom_components/alarmo/frontend/localize/languages/da.json:127]: Same comment as before

**Review by nielsfaber — CHANGES_REQUESTED** [review]
Thank you for your contribution.
Could you please make a small correction?
I will make sure this translation is added to the next release.

File: custom_components/alarmo/frontend/localize/languages/da.json (hunk 1 above)
Comment (knsjensen) [reply custom_components/alarmo/frontend/localize/languages/da.json]: I just added the change to the pull request.

File: custom_components/alarmo/frontend/localize/languages/da.json (hunk 2 above)
Comment (knsjensen) [reply custom_components/alarmo/frontend/localize/languages/da.json]: I just added the change to the pull request.

File: custom_components/alarmo/frontend/localize/languages/da.json (hunk 3 above)
Comment (knsjensen) [reply custom_components/alarmo/frontend/localize/languages/da.json]: I just added the change to the pull request.

**Pull request merged**